paasta_tools/api/api_docs/swagger.json

```diff
@@ -1295,6 +1295,10 @@
                     "type": "string",
                     "description": "name of the pod in Kubernetes"
                 },
+                "host": {
```

Comment:
This is from another PR, right? rebase maybe?

Reply:
No it isn't the swagger spec is out of date.